ext/mbstring/tests/gh10192_utf7.phpt

```diff
@@ -0,0 +1,244 @@
+--TEST--
+GH-10192 (mb_detect_encoding() results for UTF-7 differ between PHP 8.0 and 8.1)
+--EXTENSIONS--
+mbstring
+--FILE--
+<?php
+
+$testcases = [
+    'non-base64 character after +' => 'A + B',
```

Comment:
Ah, this is great. Thanks very much.

I still need to carefully examine the `--EXPECT--` section and make sure that the expected results make sense for each test case.